contribution/development_guide.md

```diff
@@ -107,6 +162,52 @@
 
       Please ensure that the steps to setup K8s is followed so as to resolve any open dependencies.
 
+2. Self-managed Kubernetes
+   * Requirements
+
+     List of minimum requirements for self-managed Kubernetes.
```

Comment:
Consistency with the description of Vagrant environment: List of -> Here is the list of